GameData/RealismOverhaul/Engine_Configs/U1250_Config.cfg

```diff
@@ -0,0 +1,286 @@
+//
+//	Dry Mass: 12.9 kg				//similar TWR to AJ10-27
+//	Thrust (SL): 19.60 kN
+//	Thrust (Vac): 23.00 kN
+//	ISP: 205.6 SL / 241.3 Vac
+//	Burn Time: 60
+//	Chamber Pressure: 2.3 MPa
+//	Propellant: AK20 / Kerosene (T-1)
+//	Prop Ratio: 3.8					//2988 Kelvin
+//	Throttle: N/A
+//	Nozzle Ratio: 5
+//	Ignitions: 1
+//	Reaction Efficieny: 92.59%
+//	=================================================================================
+//	Sources:	https://ru.wikipedia.org/wiki/МР-1
+//				https://www.airplanesandrockets.com/rockets/soviet-mr-1-meteo-rocket-march-1967-american-modeler.htm
+
+//	Used by:
+
+//	Notes:
+//	Stats are mostly guessed sanity checked by Rocket Propulsion Analysis, there's very little solid information.
+//	===============================================================
+
+@PART[*]:HAS[#engineType[U1250]]:FOR[RealismOverhaulEngines]
+{
+	%title = U-1250
+	%manufacturer = OKB-2 (Isayev)
+	%description = Small sustainer for MR-1 Meteo sounding rockets, developed in response to the American Aerobee. Pressure-fed. Used after a small solid booster.
+
+	%specLevel = operational
+
+	@MODULE[ModuleEngines*]
+	{
+		%EngineType = LiquidFuel
+	}
+
+	!MODULE[ModuleEngineConfigs],*{}
+	!MODULE[ModuleGimbal] {}
+	!MODULE[ModuleAlternator],*{}
+	!RESOURCE,*{}
+
+	MODULE
+	{
+		name = ModuleEngineConfigs
+		type = ModuleEngines
+		origMass = 0.0147
+		configuration = U-1250
+		modded = false
+		
+		CONFIG
+		{
+			name = U-1250
+			description = Isayev engine that powered the Soviet MR-1 sounding rocket.
+			specLevel = operational
+			maxThrust = 14.417
+			minThrust = 14.417
+			massMult = 1.0
+			heatProduction = 40
+			autoVariationScale = 1.5
+
+			PROPELLANT
+			{
+				name = Kerosene
+				ratio = 0.32714
+				DrawGauge = True
+			}
+
+			PROPELLANT
+			{
+				name = AK20
+				ratio = 0.67286
+			}
+
+			PROPELLANT
+			{
+				name = Nitrogen
+				ratio = 31.5
+				ignoreForIsp = true
+			}
+
+			atmosphereCurve
+			{
+				key = 0 232.1
+				key = 1 204.8
+			}
+
+			IGNITOR_RESOURCE
+			{
+				name = Tonka250
+				amount = 1
+			}
+
+			ullage = True
+			ignitions = 1
+			pressureFed = True
+		}
+		CONFIG
+		{
+			name = U-1700
+			description = Speculative upgrade for the U-1250
+			specLevel = altHist
+			maxThrust = 19.411
+			minThrust = 19.411
+			massMult = 1.0
+			heatProduction = 40
+			autoVariationScale = 1.3
+
+			PROPELLANT
+			{
+				name = Kerosene
+				ratio = 0.32196
+				DrawGauge = True
+			}
+
+			PROPELLANT
+			{
+				name = AK20
+				ratio = 0.67804
+			}
+
+			PROPELLANT
+			{
+				name = Nitrogen
+				ratio = 33.0
+				ignoreForIsp = true
+			}
+
+			atmosphereCurve
+			{
+				key = 0 236.4
+				key = 1 206.5
+			}
+
+			IGNITOR_RESOURCE
+			{
+				name = Tonka250
+				amount = 1
+			}
+
+			ullage = True
+			ignitions = 1
+			pressureFed = True
+		}
+		CONFIG
+		{
+			name = U-2000
+			description = Sustainer engine for the R-112 SAM. Tested, but never entered service.
+			specLevel = operational
```

Comment:
Last thing, I think.
Is this "operational" or "prototype" based on the comment?

Reply:
They appear to have launched a few of them off, so operational, if only briefly.